scico/linop/radon_svmbir.py

```diff
@@ -39,6 +41,7 @@ def __init__(
         input_shape: Shape,
         angles: np.ndarray,
         num_channels: int,
+        is_masked=False,
     ):
         """
         Args:
```

Comment:
More explanation on the arguments would be nice

Reply:
Done